### ISSUE TITLE
AnalyticalTable how to use reactTableOptions

### ISSUE THREAD

**Issue by nazarkulyk** [SAP/ui5-webcomponents-react#1494]
I am trying to find out how to use **reactTableOptions** property. What i want to achieve is to make Input field outside analyticaltable which will be global filter for the table. That should be possible as soon as i see, there is **useTable** hook with **useGlobalFilter** based on **react-table** in webcomponent. But i dont get how to use it?

Some little bit more advanced example for **reactTableOptions** on StoryBoard would help with it a lot.

Also its not clear how to set **initalState** for **react-table**. As example if i have already some column filter at first rendering. How can i do it?

**Comment by nazarkulyk** [comment]
Found some way how to set initial filter:

```javascript
   reactTableOptions={{
        initialState: { filters: [{ id: 'name', value: tableFilterValue }] }        
      }}
```

**Comment by Lukas742** [comment]
Hi @nazarkulyk 

we improved the table to now also accept a `globalFilterValue` prop, where you can pass the value you want to search on all columns. You can disable it for specific columns by setting `disableGlobalFilter` to the respective column in the columns array. 
Furthermore we added a section in the documentation where it's explained how to implement custom plugin hooks.

I will let you know once this is released.

**Comment by Lukas742** [comment]
Released in [0.15.4](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v0.15.4)
